ipfs-cluster-ctl/main.go

```diff
@@ -128,19 +129,17 @@ This command will print out information about the cluster peer used
 			},
 		},
 		{
-			Name:  "peers",
-			Usage: "list and manage IPFS Cluster peers",
-			UsageText: `
-This command can be used to list and manage IPFS Cluster peers.
-`,
+			Name:        "peers",
+			Description: "list and manage IPFS Cluster peers",
 			Subcommands: []cli.Command{
 				{
 					Name:  "ls",
 					Usage: "list the nodes participating in the IPFS Cluster",
-					UsageText: `
-This commands provides a list of the ID information of all the peers in the Cluster.
+					Description: `
+This command provides a list of the ID information of all the peers in the Cluster.
 `,
-					Flags: []cli.Flag{parseFlag(formatID)},
+					Flags:     []cli.Flag{parseFlag(formatID)},
+					ArgsUsage: " ",
```

Comment:
Probably doesn't make a lot of difference, but `ArgsUsage: ""` instead of `" "`? Or is there a reason for the space?

Hm, after looking further on it seems you do this a few times. I won't mark all of them, it's starting to look intentional.